### PR TITLE
fixed loading issue 

### LINKED ISSUE #137: Bug while Loading
**Describe the bug**

**To Reproduce**

1. Select a 'state.'
2. Select a 'district'
3. Scroll down to pagination

**Expected behavior**

The data returned is not sliced to the point we require!

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
null

### PR DESCRIPTION
**Pull request by Justinnn07**
> 🚨 Please review the [guidelines for contributing](../CONTRIBUTING.md) to this repository.

### Please check if the PR fulfills these requirements

- [x] Make sure you are requesting to **NEW-UI**. Don't request other protected Branches like **staging/master**
- [x] Make sure no conflicts are present in the code, if so please resolve it(_Tip: Always fetch upstream_)
- [x] Your Commit messages should make sense.
- [x] Don't push your package.lock.json as this project uses yarn.lock already.
- [x] Check your code additions will fail neither code linting checks nor unit test.

### Describe your changes

- **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- Bug 

- **What is the current behavior?** (You can also link to an open issue here)
    resolves #137 

- **Other Information**:

❤️ Thank you!

### CODE REVIEW

File: src/components/VaccineData/VaccineDataMain.js [hunk 1]
```diff
@@ -1,29 +1,14 @@
-import VaccineDataSingle from './SingleVaccineData/VaccineDataSingle';
-import CircularProgress from '@material-ui/core/CircularProgress';
+import VaccineDataSingle from "./SingleVaccineData/VaccineDataSingle";
 
 const VaccineDataMain = ({ vaccineData, loading }) => {
```
Comment (stephin007) [inline src/components/VaccineData/VaccineDataMain.js:3]: Loading is decalred but not used? 
Why though?

**Comment by stephin007** [comment]
@Justinnn07 left few comments!

File: src/components/VaccineData/VaccineDataMain.js (hunk 1 above)
Comment (Justinnn07) [reply src/components/VaccineData/VaccineDataMain.js]: I need to fix the loading issue manually , so switched to Home!

**Comment by stephin007** [comment]
@Justinnn07 again left some review comments

**Comment by stephin007** [comment]
@Justinnn07 why is the prop removed in line 22 of nullState.js?

Could you please explain!

**Review by stephin007 — COMMENTED** [review]
@Justinnn07 left some comments

**Comment by Justinnn07** [comment]
> @Justinnn07 why is the prop removed in line 22 of nullState.js?
> 
> Could you please explain!

the prop `vaccineData` is removed due to multiple passing of data from Home.js, which will replicate the vaccine info .. 

Let me know if there is any other changes!

File: src/components/NullState.js [hunk 2]
```diff
@@ -11,15 +11,15 @@ const NullState = ({
...
+    toSearchValue === "Find By District" ||
+    toSearchValue === "Find By District & Date(Slots for next 7 days)"
   ) {
-    if (districtCode === 'PLEASE SELECT A STATE FIRST') {
+    if (districtCode === "PLEASE SELECT A STATE FIRST") {
       return <div></div>;
     } else if (vaccineData.length > 0) {
       return (
         <div>
-          <VaccineDataMain vaccineData={vaccineData} />
```
[the remark this answers is not in the document]
Comment (Justinnn07) [reply src/components/NullState.js:22]: the prop `vaccineData` is removed due to multiple passing of data from Home.js, which will replicate the vaccine info ..

Let me know if there is any other changes!

**Review by stephin007 — APPROVED** [review]
Looks Good 🥇

**Pull request merged**